web-console/package.json

```diff
@@ -35,8 +35,7 @@
   "scripts": {
     "compile": "./script/build",
     "pretest": "./script/build",
-    "run": "./script/run",
-    "test": "npm run tslint && npm run stylelint && jest --silent 2>&1",
+    "test": "npm run tslint && npm run stylelint && npm run check-licenses && jest --silent 2>&1",
```

Comment:
It looks like this causes the license checker to be run twice during CI, first by maven independently https://travis-ci.org/apache/incubator-druid/jobs/561206509#L1293, and then by side effect when maven runs test. We could probably remove the `check-licenses` maven execution from the web-console pom.xml, https://github.com/apache/incubator-druid/blob/master/web-console/pom.xml#L67, though it's probably not a huge deal to leave in?